MantidQt/CustomInterfaces/src/MantidEVWorker.cpp

```diff
@@ -156,12 +156,21 @@ bool MantidEVWorker::loadAndConvertToMD(
       IAlgorithm_sptr alg = AlgorithmManager::Instance().create("Load");
       alg->setProperty("Filename", file_name);
       alg->setProperty("OutputWorkspace", ev_ws_name);
-      alg->setProperty("Precount", true);
+      alg->setProperty("FilterByTofMin", 1000.0);
```

Comment:
I could add limits to interface, but this is something they have never wanted to change until these TOF=0's appeared. I will use your cutoff suggestion.